src/components/HorizontalProgressBar/scss/HorizontalProgressBar.scss

```diff
@@ -0,0 +1,20 @@
+$bar-height: 20px;
+$bar-border-radius: 5px;
+$default-bar-clr: #af1e2d;
+$default-background-clr: #666666;
+.horizontal-progress-bar {
+  position: relative;
+  background-color: $default-background-clr;
+  height: $bar-height;
+  border-radius: $bar-border-radius;
+}
+
+
+.horizontal-progress-bar__progress {
+  position: absolute;
+  left: 0;
+  top: 0;
+  height: 100%;
+  background-color: $default-bar-clr;
+  border-radius: $bar-border-radius;
```

Comment:
i think this would be more smooth if we use `translateX` instead of `width` - use that yummy graphics card that people have.

Reply:
so I added some css animation. it will progressively translate to its position on load
```
.horizontal-progress-bar__progress {
  animation: progress-animation ease-in 0.4s;
...
}

@keyframes progress-animation {
  0% {
    transform: translateX(-100%);
  }
  100% {
    transform: translateX(0);
  }
}
```